api/api.go

```diff
@@ -274,6 +273,41 @@ func (c *Client) RequestToJoinTeam(
 	return nil
 }
 
+// ListRequestsToJoinTeam for the team with the given UUID.
+func (c *Client) ListRequestsToJoinTeam(teamUUID uuid.UUID, fingerprint fpr.Fingerprint) (
+	requestsToJoinTeam []team.RequestToJoinTeam, err error) {
+
+	path := fmt.Sprintf("team/%s/requests-to-join", teamUUID)
+	request, err := c.newRequest("GET", path, nil)
+	if err != nil {
+		return nil, err
+	}
+	request.Header.Add("authorization", authorization(fingerprint))
+	decodedJSON := new(v1structs.ListRequestsToJoinTeamResponse)
+	_, err = c.do(request, &decodedJSON)
+	if err != nil {
+		return nil, err
+	}
+	for _, jsonRequestToJoin := range decodedJSON.Requests {
+		requestUUID, err := uuid.FromString(jsonRequestToJoin.UUID)
+		if err != nil {
+			continue
+		}
+		requestFingerprint, err := fpr.Parse(jsonRequestToJoin.Fingerprint)
```

Comment:
One day I'd like to change the `v1structs` so that fingerprint is a `fpr.Fingerprint` rather than a `string`, then all of this parsing and formatting will just work automatically... imagine!